apps/www/src/registry/default/plate-ui/link-floating-toolbar.tsx

```diff
@@ -67,11 +74,15 @@ export function LinkFloatingToolbar({ state }: LinkFloatingToolbarProps) {
     editButtonProps,
     unlinkButtonProps,
   } = useFloatingLinkEdit(editState);
+  const inputProps = useFormInputProps({
+    preventDefaultOnEnterKeydown,
+    onKeyDownCaptureCallback,
+  });
 
   if (hidden) return null;
 
   const input = (
-    <div className="flex w-[330px] flex-col">
+    <div className="flex w-[330px] flex-col" {...inputProps}>
```

Comment:
@PaulSinghDev Sorry for the confusion, here is how a consumer could provide their own `onKeyDownCapture`:

```ts
<div className="flex w-[330px] flex-col" {...inputProps} onKeyDownCapture={(e) => {
  inputProps.onKeyDownCapture(e)
  customOnKeyDownCapture(e)
  ...
```

Setting `preventDefaultOnEnterKeydown: true` is just a default value the user can customize on his end

Reply:
@zbeyens No drama at all! I think I get you now, thanks for clarifying. With this, I'm guessing the suggestion is the user can alter the contents of `<LinkFloatingToolbar />` themselves since we are following the ShadCN pattern of 'we give you the base, you change how you see fit'. I've just updated the code and the unit test to reflect these changes and will push now

Reply:
This is indeed the goal, maintaining props is otherwise a hell :)

Reply:
Nice one mate! I think I have addressed these issues in the latest commit. Sorry again I've been a bit dim ha